crates/biome_js_analyze/tests/specs/nursery/useExportsLast/valid_multiple_separate_exports.js

```diff
@@ -0,0 +1,11 @@
+const arr = ["bar"];
+
+export const bool = true;
+
+export default bool;
+
+export function func() {
+	console.log("Hello World 🌍");
+}
+
+export const str = "foo";
```

Comment:
This example code was taken directly from one of the examples at https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md.  That code is MIT licensed and I want to provide proper attribution and could use a little guidance to make sure I get it right.

Reply:
No problem to add some modification